backend/backend/urls.py

```diff
@@ -55,6 +57,9 @@
 	# Match Views
 	path('match/', MatchView.as_view()),
 	path('match/test/', LaunchTestMatchView.as_view()), # TEMPORARY
+
+	# Stats Views
+	path('stats/', StatsView.as_view()),
```

Comment:
So, what stats exactly does this return? Some fake data so far? I think we should separate different stats / data / graphs to their own routes. Like one route to return a graph / data formed from wins and another one from loses or something. But, you know better. For now I dont see any actual functionality here so its a little hard to say whats being added.

Reply:
At this point yes fake data. I will add authentication and delivery of user's stats in the upcoming PRs. And in future PRs the url will also change into /stats/<int:user_id>